.env.example

```
@@ -0,0 +1,4 @@
+DATABASE_URL="postgresql://user:password@localhost:5432/mydb?schema=public"
```

dotenv-linter (3.3.0)

[warning] [QuoteCharacter] The value has quote characters (', ")

(QuoteCharacter)